### PR TITLE
chore(loader): Releasing 1.1.8 to fix connect kit version

### PR DESCRIPTION
**Pull request by ComradeAERGO**

### CODE REVIEW

File: packages/connect-kit-loader/CHANGELOG.md [hunk 1]
```diff
@@ -7,22 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 1.1.8 - 2023-12-15
+
+Fixed version of connect-kit to 1.1.8
```
Comment (Justkant) [inline packages/connect-kit-loader/CHANGELOG.md:12]:
```suggestion
## 1.1.3 - 2023-12-15

Fixed version of connect-kit to 1.1.8
```

File: packages/connect-kit-loader/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ledgerhq/connect-kit-loader",
   "sideEffects": false,
-  "version": "1.1.2",
+  "version": "1.1.8",
```
Comment (Justkant) [inline packages/connect-kit-loader/package.json:4]:
```suggestion
  "version": "1.1.3",
```

**Review by Justkant — CHANGES_REQUESTED** [review]
We can just use `1.1.3` for the new loader version IMO

**Review by Justkant — APPROVED** [review]

**Comment by ComradeAERGO** [comment]
Keeping 1.1.8 to sync with the connect-kit version it's tethered to

**Pull request merged**